numpy/_core/einsumfunc.py

```diff
@@ -714,6 +714,9 @@ def _parse_einsum_input(operands):
 
     # Make sure output subscripts are in the input
     for char in output_subscript:
+        if output_subscript.count(char) != 1:
```

Comment:
I don't recall the exact logical pathways. If we have moved this logic here- can we remove the duplicate logic from `np.einsum` which is causing the non-uniform error responses?